sechub-doc/src/docs/asciidoc/documents/code2doc/client/02_sechub_client.adoc

```diff
@@ -84,46 +84,44 @@ Following actions are supported:
 
 ===== scan
 
-Does scan synchronous which means
+Does a synchronous scan which means
 
- - will create a `{sechub}` job
+ - create a `{sechub}` job
  - upload necessary stuff to `{sechub}` (e.g. zipped sourcefiles for source scans)
- - approve job as being ready to start
- - wait for job done,
- - fetch automatically result report to output folder (html or json)
- - returns non-zero on `red` so your build will break on critical or high findings (depends on your build tool)
+ - approve job as ready to start
+ - wait until the job is done,
+ - download the report file
+ - exits non-zero on `red` (critical or high findings) so your build will break (depends on your build tool)
 
-IMPORTANT: For security reasons always pass your API-token via the environment variable `SECHUB_APITOKEN`.
+This is the recommended scan action: You only have to make one single client call to trigger a scan and get the report.
 
-TIP: The is the most preferred scan action for builds - you only have to make one
-     single client call and your are done.
+IMPORTANT: For security reasons always pass your API-token via the environment variable `SECHUB_APITOKEN`.
 
 **Minimum call syntax**
 
 Prerequisite is a configuration file `sechub.json` in current directory. +
-See here for details: <<section-client-configuration-file,Configuration file>>
+See here for details: <<section-client-configuration-file,Configuration file>> changeme!!!
```

Comment:
What does "changeme!!!" mean?

Reply:
:smile: That was a marker in case the link reference would be different after editing.
In fact ist has not changed.
-> removed the comment